src/hypersync/RestDataSource.ts

```diff
@@ -121,6 +121,7 @@ export class RestDataSource extends ApiClient implements IDataSource {
   public async getData<TData>(
     dataSetName: string,
     params?: DataValueMap,
+    page?: string,
```

Comment:
Everywhere else, these parameters are listed in this order: page, metadata. I wanted to preserve that consistency, even though technically, this makes it a breaking change. Going to version 1.0.0 over this seems like overkill though

Reply:
We are not yet using semver as we should be.  This is OK.